src/operator/tensor/broadcast_reduce_op_value.cc

```diff
@@ -284,10 +284,16 @@ array.
 
 Examples::
 
-  x = [[1, 2],
-       [3, 4]]
+  x = [[[1,2],
+        [3,4]],
+       [[2,2],
+        [5,6]]]
 
-  norm(x) = [5.47722578]
+  norm(x, ord=2, axis=1) = [[3.1622777 4.472136 ]
+                            [5.3851647 6.3245554]]
+
+  norm(x, ord=1, axis=1) = [[4., 6.],
+                            [7., 8.]]
 
   rsp = x.cast_storage('row_sparse')
```

Comment:
Please also add specification for sparse-related cases below.

Reply:
I do not understand what you mean by "specification for sparse-related cases" could you please elaborate?

Reply:
the lines below have examples for sparse, and you should update them with the latest changes to the behavior of this op, you should also add that only "ord=2" supports sparse ndarrays after your changes.

Reply:
fixed